src/library_eventloop.js

```diff
@@ -82,14 +80,12 @@ LibraryJSEventLoop = {
         if (index >= 0 && index < __setImmediate_queue.length) __setImmediate_queue[index] = () => {};
       })
     }`,
+  $emSetImmediate: undefined,
 
-  $polyfillSetImmediate: () => {
```

Comment:
not sure I really understand; is this deduplication not necessary anymore? or just not worth the complexity?